src/pio_usb.c

```diff
@@ -97,7 +97,7 @@ void __not_in_flash_func(pio_usb_bus_usb_transfer)(pio_port_t *pp,
     continue;
   }
   pp->pio_usb_tx->irq = IRQ_TX_ALL_MASK; // clear complete flag
-  while (*pc <= PIO_USB_TX_ENCODED_DATA_COMP) {
+  while (*pc < PIO_USB_TX_ENCODED_DATA_COMP) {
```

Comment:
@tannewt  not sure if needed, but this follow original author comment, let me know if you think we should revert this.